src/components/TabBar.js

```diff
@@ -27,12 +27,6 @@ function TabPanel(props) {
   );
 }
 
-TabPanel.propTypes = {
-  children: PropTypes.node,
-  index: PropTypes.number.isRequired,
-  value: PropTypes.number.isRequired,
-};
-
 function a11yProps(index) {
```

Comment:
Interesting

Reply:
That will be the next issue to work on haha.